scrapi/settings/consumerManifests/arxiv_oai.json

```diff
@@ -0,0 +1,10 @@
+{
```

Comment:
I think that may be a bit out of scope for this PR, since it would require changing how running the consumers works